### PR TITLE
[zh-hans] Align ecto/changeset and add zh-hans contributors

### PR DESCRIPTION
**Pull request by kenspirit**

### CODE REVIEW

File: _data/contributors.yml [hunk 1]
```diff
@@ -400,3 +400,23 @@
   homepage: https://kate-travers.com/
   contributions:
     - blog
+
+- github: zillou
```
Comment (doomspork) [inline _data/contributors.yml]: Thank you for doing this ❤️ 

Could you remove `intro`, `homepage`, and `twitter` as long as they are empty? I believe that will cause some issues.

Comment (kenspirit) [reply _data/contributors.yml]: It seems there are many other contributor section also have blank field as well, should I remove them all?

Comment (doomspork) [reply _data/contributors.yml]: Apparently we handle this in the templates, thanks for pointing that out @kenspirit 😁

File: zh-hans/lessons/ecto/changesets.md [hunk 2]
```diff
@@ -213,7 +213,7 @@ end
 iex> User.changeset(%User{}, %{"name" => "Bob"})
 #Ecto.Changeset<
   action: nil,
-  changes: %{first_name: "Bob"},
+  changes: %{name: "Bob"},
```
Comment (doomspork) [inline zh-hans/lessons/ecto/changesets.md]: Care to do another PR applying this change to all languages impacted?

Comment (kenspirit) [reply zh-hans/lessons/ecto/changesets.md]: Will work on this after work tonight.
